SPECS/sed/sed.spec

```diff
@@ -17,7 +16,7 @@ The Sed package contains a stream editor.
 %package lang
 Summary: Additional language files for sed
 Group: System Environment/Programming
-Requires: sed >= 4.5
+Requires: sed >= 4.8
```

Comment:
Do you know, if we could use `>= %{version}` instead?

Reply:
Yes I believe so, will update